### DISCUSSION TITLE
Location of mc after installing from mc.deb

### DISCUSSION THREAD

**Discussion by binaryfire** [minio/minio#18883] [Q&A]
Hi guys,

I'm automating things with Ansible, so installing from a deb makes it easier to achieve idempotency. I've installed the client from `https://dl.min.io/client/mc/release/linux-amd64/mc.deb` but I can't find where it put the binary :) Could someone help?

**Comment by binaryfire** [comment]
Just realised it installed it as `mcli` (usr/local/bin/mcli) instead of `mc`. Any reason it's named differently in the deb install?

**Reply by harshavardhana** [reply]
That must be fixed.

**Reply by binaryfire** [reply]
I’ll open an issue

**Reply by SkyWriter** [reply]
The reason is in both Minio Client struggling to take over the binary name of a 30 year old (happy anniversary!) widely adopted Midnight Commander, and OS maintainers reasonably refusing to accept it. This must be fixed. By finally giving Minio Client a unique name of its own.
